### PR TITLE
DS-2079- modified scheduling pipeline run module to include numerica…

### PR DESCRIPTION
**Pull request by chtyler**
…l frequency value for periodic runs

## Description
Documented that the user can add a numerical input for scheduled runs

## How Has This Been Tested?
Tested against recent build of ODH. 

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: modules/scheduling-a-pipeline-run.adoc [hunk 1]
```diff
@@ -33,7 +33,7 @@ The *Create run* page opens.
 * Select *Run once immediately after creation* to specify the run executes once, and immediately after its creation.
 * Select *Schedule recurring run* to schedule the run to recur.
 ... Configure the run's trigger type.
-.... Select *Periodic* and select the execution frequency from the list.
+.... Select *Periodic* to specify an execution frequency. Enter a numerical value and select an execution frequency from the list.
```
Comment (MelissaFlinn) [inline modules/scheduling-a-pipeline-run.adoc:36]: please clarify - line 36 - should you changed **and** to **or**? 
That is, the either type in a value of select one from the list (they don't do both, right?)

Comment (chtyler) [reply modules/scheduling-a-pipeline-run.adoc:36]: It's both. They need to type in a value and select an appropriate frequency from the list.

For example, entering 10 and selecting hours from the list executes a pipeline run every 10 hours.

Comment (MelissaFlinn) [reply modules/scheduling-a-pipeline-run.adoc:36]: thanks for explaining - I think it will be obvious to the user when looking at the UI

**Comment by MelissaFlinn** [comment]
lgtm

**Review by diegolovison — APPROVED** [review]

**Pull request merged**